File: fieldsync-inventory-backend/src/main/java/com/example/fieldsync_inventory_api/dto/stock/movement_type/StockMovementTypeResponseDTO.java
package com.example.fieldsync_inventory_api.dto.stock.movement_type;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import com.example.fieldsync_inventory_api.enums.EffectOnStock;

import java.time.Instant;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class StockMovementTypeResponseDTO {
    private Integer id;
    private String name;
    private EffectOnStock effectOnStock;
    private String description;
    private Instant createdAt;
    private Instant updatedAt;
    private Instant deletedAt;
}

File: fieldsync-inventory-app/app/src/main/java/com/example/fieldsync_inventory_app/data/remote/dto/stock_movement_type/StockMovementTypeResponseDto.kt
package com.example.fieldsync_inventory_app.data.remote.dto.stock_movement_type

data class StockMovementTypeResponseDto(
    val id: Int,
    val name: String,
    val effectOnStock: String,
    val description: String?,
    val createdAt: String,
    val updatedAt: String,
    val deletedAt: String?
)